### PR TITLE
Origin/feature/tasks/main activity

### PR DESCRIPTION
**Pull request by DPetrushevski6689**

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
     android:layout_height="match_parent"
+    android:orientation="vertical"
     tools:context=".MainActivity">
 
-    <TextView
-        android:layout_width="wrap_content"
+    <com.google.android.material.textfield.TextInputLayout
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
```
Comment (Andrejpopof) [inline app/src/main/res/layout/activity_main.xml]: I WANT THIS TO STAY . DONT DELETE IT IDIOT I AM THE MANAGER STUPID

Comment (Andrejpopof) [reply app/src/main/res/layout/activity_main.xml]: DONT SEND ME THIS BS . YOU ARE FIRED

Comment (Andrejpopof) [reply app/src/main/res/layout/activity_main.xml]: GO WORK IN YOUR MOTHER BASEMENT YOU SLAVE

Comment (DPetrushevski6689) [reply app/src/main/res/layout/activity_main.xml]: :(

**Pull request merged**